content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -626,8 +631,8 @@ case.
 
 With the Server Side Apply feature enabled, the `PATCH` endpoint accepts the
 additional `application/apply-patch+yaml` content type. Users of Server Side
-Apply can send partially specified objects to this endpoint. An applied config
-should always include every field that the applier has an opinion about.
+Apply can send partially specified objects as yaml or json to this endpoint.
```

Comment:
That's exactly what we're trying to clarify though. We've had several people ask us "You have a YAML endpoint, but don't have a json endpoint, what am I supposed to do?"

Reply:
The rewording I suggest at https://github.com/kubernetes/website/pull/19286#discussion_r392344512 aims to help exactly those people.

I wouldn't tell them that the API server accepts JSON labelled as `application/apply-patch+yaml` because that almost sounds like a bug; I would teach them that if they have JSON then that is also a valid YAML document and they can use that at an endpoint expecting them to send YAML.

Reply:
Yeah, that's good, thanks!